long-ride-alerts/src/test/java/org/apache/flink/training/exercises/longrides/LongRidesUnitTest.java

```diff
@@ -110,6 +115,16 @@ public void shouldAlertWithStartFirst() throws Exception {
         assertThat(resultingRideId()).isEqualTo(rideStarted.rideId);
     }
 
+    @Test
+    public void shouldThrowExceptionWhenIsStartFalse() throws Exception {
+        exceptionRule.expect(LongRidesException.class);
```

Comment:
Sorry, what we are checking here?
As I see you introduced variable
```
    @Rule
    public ExpectedException exceptionRule = ExpectedException.none();
```
a now just do this, what is the reason of it?

Reply:
checking if the method is throwing an exception when startTime is not available in the end event

Reply:
Please use proper methods for it
https://github.com/junit-team/junit5/blob/main/junit-jupiter-api/src/main/java/org/junit/jupiter/api/AssertThrows.java#L41
thanks

Reply:
Rewritten the Junit to handle exception using Junit-5